about.html

```diff
@@ -0,0 +1,154 @@
+            <span class="l-1 bar"></span>
+            <span class="l-2 bar"></span>
+            <span class="l-3 bar"></span>
+          </div>
+        </div>
+      </div>
+      <section class="headlines">
+        <div class="advert-txt">
+          <h1>Creative Idea Discustions 2022</h1>
+        </div>
+        <div class="box-txt">
+          <p>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Iure, ad
+            necessitatibus nesciunt voluptate numquam odit veniam reprehenderit
+            expedita doloremque laboriosam natus eligendi itaque, maxime
+            corporis dolorum ipsa neque deserunt. Voluptatem. Lorem ipsum dolor
+            sit amet consectetur adipisicing elit. Iure, ad necessitatibus
+            nesciunt voluptate numquam odit veniams
+          </p>
+        </div>
+        <div class="address-txt">
+          <p>
+            Please contact us per email for further quetsions about our campany!
+          </p>
+          <br />
+          <a href="#">kumilachew1212@gmail.com</a>
+        </div>
+        <div class="logo-container">
+          <h3 class="logo-desk">Our History</h3>
+          <p class="logo-para">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Iure, ad
+            necessitatibus nesciunt voluptate numquam odit veniam reprehenderit
+            expedita doloremque laboriosam natus eligendi itaque, maxime
+            corporis dolorum ipsa neque deserunt. Voluptatem. Lorem ipsum dolor
+            sit amet consectetur adipisicing elit. Iure, ad necessitatibus
+            nesciunt voluptate numquam odit veniams
+          </p>
+          <div class="logo-detail logo-desk">
+            <img
+              src="./images/eagle-logo.jpeg"
+              alt=""
+              class="logo-img img-small"
+            />
+            Haddis Tecnology
+          </div>
+          <hr />
+          <div>
+            <h3 class="logo-desk">Lorem ipsum dolor sit amet consectetur</h3>
+            <p class="logo-desk">
+              Take a look at the two cc global submmits which took place in
+              buenos aires and in what..
+            </p>
+            <div class="complete">
+              <article class="img-main">
+                <div class="img-list">
+                  <img class="img-list" src="/images/web-3.jpeg" alt="" />
+                </div>
+                <div class="system-txt">
+                  <h4 class="year">2021</h4>
+                  <p class="txt-1">Student Registration System</p>
+                </div>
+              </article>
+              <article class="img-main">
+                <div class="img-list">
+                  <img class="img-list" src="/images/web-2.jpeg" alt="" />
+                </div>
+                <div class="system-txt">
+                  <h4 class="year">2022</h4>
+                  <p class="txt-1">Property Management System</p>
+                </div>
+              </article>
```

Comment:
- [ ] Please ensure that the text is aligned centrally in the past conference images like in the Behance design

> Behance
![image](https://user-images.githubusercontent.com/90556221/181484483-3ccb1e98-b444-4604-a817-04af72056ebe.png)

> Yours
![image](https://user-images.githubusercontent.com/90556221/181484593-b4d4d958-eb27-460d-9cfd-94363e4b157d.png)